example/capsnet/capsulenet.py

```diff
@@ -255,30 +273,53 @@ def random_shift(x, width_shift_fraction, height_shift_fraction):
     x = apply_transform(x, shift_matrix, 'nearest')
     return x
 
+
 def _shuffle(data, idx):
     """Shuffle the data."""
```

Comment:
At times single line comment are used and at times multi-line comment are used even if the actual comment is just one line. Need to establish one way out of the two

Reply:
You are right, I think the style of comments should align to multi-line comment.